python/setup.py

```diff
@@ -30,6 +30,17 @@
 if not LIB_PATH:
     raise RuntimeError('libdlr.so missing. Please compile first using CMake')
 
+#user data consent notification
+CALL_HOME_USR_NOTIFICATION = """\n CALL HOME FEATURE ENABLED
+                            \n Your device information and model load/execution count metric data being sent to a server. \
+                            \n You can disable this feature by using below configuration steps. \
+                            \n\t1. Create a config file, ccm_config.json in your application path. \
+                            \n\t2. Added below format content in it, \
+                            \n\t\t{\n\t\t\t"ccm" : "false"\n\t\t} \
+                            \n\t3. Restart DLR application."""
+
+print(CALL_HOME_USR_NOTIFICATION)
```

Comment:
@mnuyens @dkasar-aws,

This won't work for pip install unless verbose flag is enabled. As Max suggested, we should display this in application level when user import DLRModel. In other words, can we revise this back to the previous revision, where in counter_mgr.py
```
        if CallCounterMgr._instance is None:
            if CallCounterMgr.is_feature_enabled():
                 print(warning_msg)
```

Reply:
Can we also move print after ```setup```? so the message show up at the bottom? in case user enable the verbose flag?

Reply:
@wuchih-amazon, @mnuyens 
As suggested above, the notification may not work in setup. We prefer to use notification in CounterMgr(as per earlier approach) and added the notification content in README.md file. This way we can resolve the issue and avoid CounterMgr module reference in Setup utils.